director/director_test.go

```diff
@@ -1760,11 +1761,6 @@ func TestRedirects(t *testing.T) {
 	})
 
 	t.Run("redirect-link-header-length", func(t *testing.T) {
-		server_utils.ResetTestState()
-		t.Cleanup(func() {
-			server_utils.ResetTestState()
-		})
-
```

Comment:
Nice that we can get rid of the inter-test cleanups here and in subsequent tests, but it also means the `viper.Set("Director.CacheSortMethod", "random")` that happens in each of these must not actually be doing anything. Can you remove them from the individual `t.Run` statements and move the config to occur once somewhere before the call to `AdvertiseOSDF()` at line 1744?